app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -160,11 +164,22 @@ object ProfileHelper {
 	}
 
 	val supportsHevc by lazy {
-		MediaTest.supportsHevc()
+		mediaTest.supportsHevc()
 	}
 
 	val supportsHevcMain10 by lazy {
-		MediaTest.supportsHevcMain10()
+		mediaTest.supportsHevcMain10()
+	}
+
+	fun supportsDts(context: Context, preferFfmpeg: Boolean): Boolean {
+		if (preferFfmpeg || Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+			return true
+		}
+
+		val audioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
+		val outputDevices = audioManager.getDevices(AudioManager.GET_DEVICES_OUTPUTS)
+
+		return outputDevices.any { it.encodings.contains(AudioFormat.ENCODING_DTS) }
```

Comment:
I'm not really familiar with the AudioManager but from my understanding this doesn't work properly. You're currently checking if _any_ of the connected devices has a DTS encoder. Not if the **currently in-use** audio device has a **DTS decoder**.

I'm also not sure if we should use `MediaCodecList` via `MediaCodecCapabilitiesTest` instead, since we do all our other codec checking with that utility class.

Reply:
> I'm also not sure if we should use MediaCodecList via MediaCodecCapabilitiesTest instead, since we do all our other codec checking with that utility class.

This indeed was the first implementation, which however didn't work as expected because the `MediaCodecList` only contains the codecs supported by the device itself, ignoring any extra codec supported by a device connected through eARC for example.